packages/snaps-jest/src/internals/simulation/methods/hooks/encryption.ts

```diff
@@ -0,0 +1,33 @@
+/**
+ * Mocks encrypting a value using a password. This is not a real encryption
+ * method, but rather is used to simulate encryption in the tests.
+ *
+ * Ideally we would use a real encryption method, such as
+ * `@metamask/browser-passworder`, but it doesn't seem to work with Node.js 18
+ * without some mocking. We can switch to a real encryption method once we
+ * drop support for Node.js 18.
```

Comment:
Should we just drop Node 18 for snaps-jest now? Or should we add some other implementation in the meantime?